std/zig/render.zig

```diff
@@ -2019,7 +2019,13 @@ fn renderTokenOffset(
 
                     const after_comment_token = tree.tokens.at(token_index + offset);
                     const next_line_indent = switch (after_comment_token.id) {
-                        Token.Id.RParen, Token.Id.RBrace, Token.Id.RBracket => indent - indent_delta,
+                        Token.Id.RParen, Token.Id.RBrace, Token.Id.RBracket => blk: {
+                            if (indent != 0) {
```

Comment:
shouldn't this be `if (indent > indent_delta)` ?

Reply:
Yes it should be actually!